webview.go

```diff
@@ -107,9 +117,13 @@ type WebView interface {
 	// Request string is a JSON array of all the arguments passed to the
 	// JavaScript function.
 	//
-	// f must be a function
-	// f must return either value and error or just error
+	// f must be a function that accepts an id string as the first argument, and
+	// then any other arguments that you want to pass from the JavaScript function.
 	Bind(name string, f interface{}) error
```

Comment:
I really wanted to change this to something like `f func(id string, params ...interface{})` but I'm not sure if that's any better to use.